src/components/tasks/card/index.tsx

```diff
@@ -53,6 +52,11 @@ const Card: FC<Props> = ({
     if (state?.isLoggedIn) {
       getTaskTags();
     }
+    if (cardDetails.assignee) {
```

Comment:
Would you want to do this only when user is logged in? If yes, then should this check `state.isLoggedIn` as well?

Reply:
I'll put it in a new useEffect.